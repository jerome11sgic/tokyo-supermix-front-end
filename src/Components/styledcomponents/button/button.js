import styled, { css } from "styled-components";
import theme from "../../../theme";
import { Button } from "antd";

export const PrimaryButton = styled(Button)`
  ${props =>
    props.primary &&
    css`
      border-radius: "0.1em";
      box-shadow: "20px 20px 1px 1px";
      border: none;
      outline: none;

      &:hover {
        color: ${theme.colors.black};
        outline: none;
        border: none;
      }
    `}
`;
