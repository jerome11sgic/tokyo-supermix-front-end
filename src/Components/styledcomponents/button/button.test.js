import React from "react";
import renderer from "react-test-renderer";
import { PrimaryButton } from "./button";

test("it works", () => {
  const tree = renderer.create(<PrimaryButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
