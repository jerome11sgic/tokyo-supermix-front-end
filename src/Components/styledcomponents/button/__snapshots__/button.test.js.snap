// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<button
  className="ant-btn sc-bdVaJa ibTDPh"
  onClick={[Function]}
  type="button"
/>
`;
